ui/uihandler.go

```diff
@@ -231,6 +264,9 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		if err != nil {
 			log.Errf("Unable to write json output for %v: %v", r.RemoteAddr, err)
 		}
+		if DoSave {
```

Comment:
Adding one more type to the request type variable I mentioned in my previous comment - stop / load / save

Reply:
well not save as save is that checkbox is in addition to load / json